spec/requests/api/v3/courses/index_spec.rb

```diff
@@ -302,6 +302,38 @@
     end
   end
 
+  describe "degree grade filter" do
```

Comment:
@darokel - [I think] all the filters have been created for the Find application, but the filters are available on both V3 and Public/V1 because both use the CourseSearchService. 

I think how and where the filters are tested are more of a fundamental / core issue around the TTAPI application in general. On the one hand you want to have test coverage but possibly not at the expense of duplication. As mentioned v3 tests the filters via E2E request specs, but Public V1 seems to have opted for unit-test-like controller specs. I'm not too sure what the rationale is for this.

So I guess Publish Register may want to decide:
1. Whether filters should be made available on Public / V1
2. Whether it wants to test each filter via the Public V1 endpoints and if yes, if this should be done via controller specs or request specs. If request specs, what is the role of controller specs etc etc

Possibly some largish questions that may not be settled in PR threads. For now though, do you think this is a blocker for merging?

Reply:
Nope, definitely not a blocker 👍